src/applications/personalization/profile/util/personal-information/personalInformationUtils.js

```diff
@@ -0,0 +1,127 @@
+import TextWidget from 'platform/forms-system/src/js/widgets/TextWidget';
+import RadioWidget from 'platform/forms-system/src/js/widgets/RadioWidget';
+
+const genderOptions = [
+  'woman',
+  'man',
+  'transgenderWoman',
+  'transgenderMan',
+  'nonBinary',
+  'preferNotToAnswer',
+  'genderNotListed',
+];
+const genderLabels = {
+  woman: 'Woman',
+  man: 'Man',
+  transgenderWoman: 'Transgender woman',
+  transgenderMan: 'Transgender man',
+  nonBinary: 'Non-binary',
+  preferNotToAnswer: 'Prefer not to answer',
+  genderNotListed: 'A gender not listed here',
+};
+const sexualOrientationOptions = [
+  'lesBianGayHomosexual',
+  'straightOrHeterosexual',
+  'bisexual',
+  'queer',
+  'dontKnow',
+  'preferNotToAnswer',
+  'sexualOrientationNotListed',
+];
+const sexualOrientationLabels = {
+  lesBianGayHomosexual: 'Lesbian, gay, or homosexual',
+  straightOrHeterosexual: 'Straight or heterosexual',
+  bisexual: 'Bisexual',
+  queer: 'Queer',
+  dontKnow: 'Don’t know',
+  preferNotToAnswer: 'Prefer not to answer',
+  sexualOrientationNotListed: 'A sexual orientation not listed here',
+};
+export const personalInformationFormSchemas = {
+  preferredName: {
+    type: 'object',
+    properties: {
+      preferredName: {
+        type: 'string',
+        pattern: '^[A-Za-z\\s]+$',
+        minLength: 1,
+        maxLength: 25,
+      },
+    },
+    required: [],
+  },
+  pronouns: {
+    type: 'object',
+    properties: {
+      pronouns: {
+        type: 'object',
+        properties: {
+          'He/him/his': { type: 'boolean' },
+          'She/her/hers': { type: 'boolean' },
+          'They/them/theirs': { type: 'boolean' },
+          'Ze/zir/zirs': { type: 'boolean' },
+          'Use my preferred name': { type: 'boolean' },
+          'Prefer not to answer': { type: 'boolean' },
+          'Pronouns not listed here': { type: 'boolean' },
```

Comment:
These will get modified into ids that will include the slashes & spaces. I know it isn't ideal, but I'm going to go ahead and give this a 👍🏽 ... maybe we can fix this by modifying the ID inside the `CheckboxWidget` in a separate PR.

Reply:
Yeah, @Mottie who owns that code? I am tempted to write a data transformation function for that purpose because I think it lives in the vets-website/src/platform/** ?

Reply:
The design system team owns the form code. They are aware.